### PR TITLE
Local Tableland chain forking

### PR DESCRIPTION
**Pull request by dtbuchholz**
## Summary

Adds docs to show how to use LT with testnet/mainnet forks. Also, bumps some deps. Closes [DOC-99](https://linear.app/tableland/issue/DOC-99/document-local-tableland-forking-feature-for-dimo-to-use)

Also, adds docs on the new `POST /query` endpoint.

### CODE REVIEW

**Comment by dtbuchholz** [comment]
Once we release the official LT version, we can publish this.

File: docs/local-tableland/fork-chain.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+title: Forking chain state
+description: Fork a live testnet or mainnet chain and use its state on Local Tableland.
+keywords:
+  - fork
+  - fork chain
+  - local tableland
+---
```
Comment (dtbuchholz) [inline docs/local-tableland/fork-chain.md:8]: This is the primary content—walks through the forking params, example test, and calls out that testnets like Mumbai should be avoided.

File: docs/local-tableland/methods.md [hunk 2]
```diff
@@ -43,6 +43,9 @@ The `LocalTableland` constructor accepts a single argument, an object with the f
 - `verbose`: Output verbose logs to stdout.
 - `silent`: Silence all output to stdout.
 - `registryPort`: Use a custom registry hardhat port, e.g., `http://127.0.0.1:8545` is the default with port `8545`, but.
+- `forkUrl`: If forking a chain, the provider URL of the testnet or mainnet chain, such as an Alchemy or Infura URL.
```
Comment (dtbuchholz) [inline docs/local-tableland/methods.md:46]: added these params to the general LT guide

File: docs/validator/api/query.md [hunk 3]
```diff
@@ -10,15 +10,42 @@ keywords:
 
 ## Endpoint
 
-`GET /query`
+The `/query` endpoint supports both `GET` and `POST` requests, returning the results of a SQL read query, along with options for formatting the response.
```
Comment (dtbuchholz) [inline docs/validator/api/query.md:13]: new POST /query docs

**Review by brunocalza — DISMISSED** [review]
The `POST /query` changes look good to me

File: docs/local-tableland/fork-chain.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+- `forkBlockNumber`: The block number to fork from, which recreates the chain state starting at that block.
+- `forkChainId`: The chain ID of the chain to fork.
+
+Once these values are set, a few things will happen:
+
+- The chain's forked state will be initialized at the starting block number.
+- Events/logs will be replayed from the forked chain, starting from the block number specified.
+- The Tableland validator will process _all_ of the Tableland registry's events/logs from the forked chain up until the forked block, and the state of the tables will be materialized.
+
+When you initially start Local Tableland, it will take a bit of time to backfill the forked chain's state. The `forkUrl` you specify for the provider will make API calls to `eth_getLogs` in batches, and the validator replays these one-by-one. Keep in mind this will continue hitting the API until the state is fully backfilled, but it _should_ be cached after the first time you run it.
```
Comment (joewagner) [inline docs/local-tableland/fork-chain.md:26]: Might be good to include that caching only works if you use the same `forkBlockNumber` value each time you start LT?

**Review by joewagner — APPROVED** [review]
I  put one inline suggestion around using the same block number to enable caching, but regardless this looks great!

**Comment by dtbuchholz** [comment]
@joewagner good idea, added!

**Review by sanderpick — APPROVED** [review]

**Pull request merged**